platform/pulpcore/app/serializers/repository.py

```diff
@@ -67,25 +67,27 @@ class ImporterSerializer(MasterModelSerializer, NestedHyperlinkedModelSerializer
     _href = DetailNestedHyperlinkedIdentityField(
         lookup_field='name', parent_lookup_kwargs={'repository_name': 'repository__name'},
     )
-
     name = serializers.CharField(
         help_text=_('A name for this importer, unique within the associated repository.')
     )
-    last_updated = serializers.DateTimeField(
-        help_text='Timestamp of the most recent update of this configuration.',
-        read_only=True
-    )
-
     feed_url = serializers.CharField(
         help_text='The URL of an external content source.',
         required=False,
     )
-
+    download_policy = serializers.ChoiceField(
+        help_text='The policy for downloading content.',
+        allow_blank=False,
+        choices=models.Importer.DOWNLOAD_POLICIES,
+    )
+    sync_mode = serializers.ChoiceField(
+        help_text='How the importer should sync from the upstream repository.',
```

Comment:
I quess I lost the battle on pulp-dev for _sync_ vs _synchronize_ but really don't see value in using this abbreviation.  Is it really that long a word to type?

Reply:
Sync vs synchronize is often the difference between 1 and 2 lines. Obviously, this is a minor concern, but single lines are much easier to scan visually, and I think it has a noticeable impact on readability. As it turns out, sync is actually a real word, [informal]. http://www.dictionary.com/browse/sync